pep-0261.txt

```diff
@@ -5,280 +5,287 @@ Last-Modified: $Date$
+   encodings (e.g. on disk or coming in from a network)
+   into logical Python objects.
+
+Encoding
+   Mechanism for representing abstract characters in terms of
+   physical bits and bytes. Encodings allow us to store
+   Unicode characters on disk and transmit them over networks
+   in a manner that is compatible with other Unicode software.
+
+Surrogate pair
+   Two physical characters that represent a single logical
+   character. Part of a convention for representing 32-bit
+   code points in terms of two 16-bit code points.
+
+Unicode string
+   A Python type representing a sequence of code points with
+   "string semantics" (e.g. case conversions, regular
+   expression compatibility, etc.) Constructed with the
+   ``unicode()`` function.
 
 
 Proposed Solution
+=================
 
-    One solution would be to merely increase the maximum ordinal 
-    to a larger value. Unfortunately the only straightforward
-    implementation of this idea is to use 4 bytes per character.
-    This has the effect of doubling the size of most Unicode 
-    strings. In order to avoid imposing this cost on every
-    user, Python 2.2 will allow the 4-byte implementation as a
-    build-time option. Users can choose whether they care about
-    wide characters or prefer to preserve memory.
+One solution would be to merely increase the maximum ordinal
+to a larger value. Unfortunately the only straightforward
+implementation of this idea is to use 4 bytes per character.
+This has the effect of doubling the size of most Unicode
+strings. In order to avoid imposing this cost on every
+user, Python 2.2 will allow the 4-byte implementation as a
+build-time option. Users can choose whether they care about
+wide characters or prefer to preserve memory.
 
-    The 4-byte option is called "wide Py_UNICODE". The 2-byte option
-    is called "narrow Py_UNICODE".
+The 4-byte option is called ``wide Py_UNICODE``. The 2-byte option
+is called ``narrow Py_UNICODE``.
 
-    Most things will behave identically in the wide and narrow worlds.
+Most things will behave identically in the wide and narrow worlds.
 
-    * unichr(i) for 0 <= i < 2**16 (0x10000) always returns a
-      length-one string.
+* unichr(i) for 0 <= i < 2**16 (0x10000) always returns a
+  length-one string.
 
-    * unichr(i) for 2**16 <= i <= TOPCHAR will return a
-      length-one string on wide Python builds. On narrow builds it will 
-      raise ValueError.
+* unichr(i) for 2**16 <= i <= TOPCHAR will return a
+  length-one string on wide Python builds. On narrow builds it will
+  raise ``ValueError``.
 
-        ISSUE 
+  ISSUE
 
-            Python currently allows \U literals that cannot be
-            represented as a single Python character. It generates two
-            Python characters known as a "surrogate pair". Should this
-            be disallowed on future narrow Python builds?
+     Python currently allows \U literals that cannot be
```

Comment:
Just noticed that the `\U` isn't rendered correctly 😅 

Will make another commit.